server/chat-plugins/helptickets.ts

```diff
@@ -1057,14 +1109,14 @@ export const commands: ChatCommands = {
 			if (!user.named) return this.popupReply(this.tr`You need to choose a username before doing this.`);
 			const banMsg = HelpTicket.checkBanned(user);
 			if (banMsg) return this.popupReply(banMsg);
+			const tickets = getTickets();
 			let ticket = tickets[user.id];
 			const ipTicket = checkIp(user.latestIp);
 			if (ticket?.open || ipTicket) {
 				if (!ticket && ipTicket) ticket = ipTicket;
 				const helpRoom = Rooms.get(`help-${ticket.userid}`);
 				if (!helpRoom) {
 					// Should never happen
-					tickets[ticket.userid].open = false;
 					writeTickets();
 				} else {
```

Comment:
This code doesn't do anything, and is also no longer possible under the current arrangement.